index.md

```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+**Source control is indispensable for working with other developers on code.**
+
+If you already use source control then this seminar can be tailored to whatever tools you are already using. I will cover best practices for using them successfully.
+
+[Get in touch](mailto:hi@tkiley.co.uk) today to learn how to use source control effectively.
+
+#### Seminar Syllabus
+
+The seminar can cover:
+
+* Why you should use source control
+* How to setup a new Git repository
+* Basic operations of Git, using the command line and a GUI
+* Resolving merge conflicts
+* How to use the "Github workflow" to enhance collaboration
+
+Software & Technologies:
+ * [Git](https://git-scm.com/){:target="_blank"} - the industry standard source control solution
+ * [Gitlab](https://about.gitlab.com/){:target="_blank"} - a widely used cloud tool for storing source code and more
+ * [Git-Cola](https://git-cola.github.io/){:target="_blank"} - a user friendly tool for interacting with Git
+ * [Kdiff3](https://kdiff3.sourceforge.net/){:target="_blank"} - a user friendly tool for resolving merge conflicts
+
+This seminar can be tweaked to use different source control software, hosting options to match your existing workflows.  Please get in touch with me and we can customise this talk to meet your needs and match your skills.
 
 ### Automated Tests
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+Automated tests automatically check code does what the developer intends. Learning to write automated tests will free the domain experts from having to constantly re-test changes by hand.
+
+**Writing automated tests will eliminate new bugs and speed up collaboration.**
+
+Automated tests are one of the most powerful tools in a developers toolkit. My motto forged from years of practical experience is _code that isn't tested does not work_ as expected
```

Comment:
Missing full-stop and it's weird that the emphasis doesn't include the whole quote.

s/does not work_ as expected/does not work as expected_.